packages/vsphere/docs: Improve logs collection documentation (#8869)
elastic/integrations: packages/vsphere/docs/README.md

## Logs

Log collection works through a syslog daemon. You need to configure the listening host address/IP (default: localhost) and port (default: 9525) in the integration. After that you need to configure vSphere to send logs to a remote syslog host and provide the configured hostname/IP and port of the Elastic Agent host.

> 

Reviewer: ```suggestion
```

> Log collection works through a syslog daemon. You need to configure the listening host address/IP (default: localhost) and port (default: 9525) in the integration. After that you need to configure vSphere to send logs to a remote syslog host and provide the configured hostname/IP and port of the Elastic Agent host.
> 

Reviewer: ```suggestion
To collect logs, a syslog daemon is used. First, you must configure the listening host/IP address (default: localhost) and host port (default: 9525) in the integration. Then, configure vSphere to send logs to a remote syslog host and provide the configured hostname/IP and port of the Elastic Agent host.
```

Author: Thanks, done!
